app/src/main/java/com/example/fitlife/MainActivity.java

```diff
@@ -21,7 +21,7 @@
 //The Home page of the application. This is used as a way to get to the other activities not supposed to have much other functionality besides that
 public class MainActivity extends AppCompatActivity {
 
-    Button profile, meal, leader, phys, friends, timely;
```

Comment:
This Will Cause Errors for the rest of the team as they do not have the latest home/landing page

Reply:
Caused an issue on my end, but I have updated my project to reflect those of teammates.

Reply:
error fixed after updating project. New landing page is now working fine